azurerm/internal/services/synapse/synapse_managed_private_endpoint_resource.go

```diff
@@ -5,13 +5,14 @@ import (
 	"log"
 	"time"
 
+	validate2 "github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/network/validate"
```

Comment:
I think `networkValidate` for this alias would be a little clearer

Reply:
agreed, I should file a Goland bug about these actually 🙃